explainer-new.md

```diff
@@ -0,0 +1,72 @@
+# Sanitizer API: New API Explainer
```

Comment:
Nit: Can/Should we replace `explainer.md` instead of setting a new one? It will be available through git history and I don't think we should need it.